config/application.rb

```diff
@@ -81,12 +81,12 @@ class Application < Rails::Application
       templates.js
       validation.js
 
-      bootstrap-complete.css
       error_pages.css
       admin.css
       mobile/mobile.css
       rtl.css
       home.css
+      color_themes/*/{desktop, mobile}.css
```

Comment:
Did you test if `bin/rake assets:precompile` generates the right files (you can get rid of them again with `bin/rake assets:clobber`)? Pretty sure that space here is wrong :)